contracts/src/Teleporter/upgrades/TeleporterUpgradeable.sol

```diff
@@ -0,0 +1,62 @@
+// (c) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// SPDX-License-Identifier: Ecosystem
+
+pragma solidity 0.8.18;
+
+import "./TeleporterRegistry.sol";
+
+/**
+ * @dev TeleporterUpgradeable provides upgrade utility for applications built on top
+ * of the Teleporter protocol by integrating with the {TeleporterRegistry}.
+ *
+ * This contract is intended to be inherited by other contracts that wish to use the
+ * upgrade mechanism. It provides a modifier that restricts access to only Teleporter
+ * versions that are greater than or equal to `_minTeleporterVersion`.
+ */
+abstract contract TeleporterUpgradeable {
+    TeleporterRegistry public immutable teleporterRegistry;
+    uint256 internal _minTeleporterVersion;
```

Comment:
Is there a reason why `_minTeleporterVersion` couldn't be `public`?

Reply:
changed to `public`